.github/workflows/tests.yml

```diff
@@ -310,6 +310,16 @@ jobs:
     needs: linting-done
     runs-on: ubuntu-latest
 
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - _: monolith
+            _db: SQLite
+
+          - _: monolith
+            postgres: Postgres
```

Comment:
I was confused that we use `_db` in one and `postgres` in another, but now I see that the underscored names are dummy variables. Can we add a comment or make a `database`=sqlite|postgres varible?

Reply:
Yes, that's fair. I'll see if the obvious `==` works, but I don't find it very clear what does and doesn't work in these templates.

Reply:
There is a partial reference here: https://docs.github.com/en/actions/learn-github-actions/expressions

Additionally, you can enable a debugging mode for GHA which might be useful, see https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging